src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -23,7 +27,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     formState: Partial<Case>;
   } = this.ctx;
 
-  const stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  let stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  if (overrideStepsContent === 1) {
+    stepsWithContent = stepsWithContentApplicant1;
+  } else if (overrideStepsContent === 2) {
+    stepsWithContent = stepsWithContentApplicant2;
```

Comment:
As above

Reply:
i guess i thought ill check for both incase app1 ever needed to view app2 answers.